_includes/devdoc/ref_p2p_networking.md

```diff
@@ -64,13 +63,14 @@ with the most recent versions listed first. (If you know of a protocol
 version that implemented a major change but which is not listed here,
 please [open an issue][docs issue].)
 
-As of Bitcoin Core 0.13.0, the most recent protocol version is 70014.
+As of Bitcoin Core 0.14.1, the most recent protocol version is 70015.
 
 | Version | Initial Release                    | Major Changes
 |---------|------------------------------------|--------------
-| 70014   | Bitcoin Core 0.13.0 <br>  | [BIP152][]: <br>• Added `sendcmpct`, `cmpctblock`, `getblocktxn`, `blocktxn` messages <br> * Added `MSG_CMPCT_BLOCK` inventory type to `getdata` message.
-| 70013   | Bitcoin Core 0.13.0 <br>  | [BIP133][]: <br>• Added `feefilter` message
-| 70012   | Bitcoin Core 0.12.0 <br>  | [BIP130][]: <br>• Added `sendheaders` message
+| 70015   | Bitcoin Core 0.13.2 <br>           | • New banning behavior for invalid compact blocks [#9026](https://github.com/bitcoin/bitcoin/pull/9026) in v0.14.0, Backported to v0.13.2 in [#9048](https://github.com/bitcoin/bitcoin/pull/9048).
```

Comment:
The protocol bump to 70015 was made in v0.14.0 ([#9026](https://github.com/bitcoin/bitcoin/pull/9026)), then backported to v0.13.2 in [#9048](https://github.com/bitcoin/bitcoin/pull/9048).
Considering there was no clear announcement in release notes, I opted for this description with references to pull requests. Happy to change it if needed.